.github/workflows/integration.yaml

```diff
@@ -297,18 +297,13 @@ jobs:
           toolchain: stable
           target: wasm32-wasi
 
-      # TODO: Remove after https://github.com/WebAssembly/wasi-testsuite/issues/49
```

Comment:
oops this PR isn't merged, yet.. I'll revert https://github.com/WebAssembly/wasi-testsuite/pull/54